qa/rpc-tests/test_framework/util.py

```diff
@@ -230,6 +231,10 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     # RPC tests still depend on free transactions
     args = [ binary, "-datadir="+datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-blockprioritysize=50000" ]
     if extra_args is not None: args.extend(extra_args)
+    if os.getenv("PYTHON_DEBUG", ""):
```

Comment:
These are useful to retain. The PYTHON_DEBUG environment variable has already been established in the test framework. Debug outputs only appear when the variable is set. In this case I have extended it to output the command line parameters of nodes. I think others will also find this useful.